tools/oio-bootstrap.py

```diff
@@ -701,6 +718,15 @@ def save_coverage():
 on_die=cry
 """
 
+template_gridinit_meta2_indexer = """
+[Service.${NS}-${SRVTYPE}-${SRVNUM}]
+group=${NS},localhost,${SRVTYPE}
```

Comment:
I suggest putting the service in the `indexer` group, so we can stop all indexers at once.